config/default.yml

```diff
@@ -637,7 +637,7 @@ RSpec/SubjectStub:
   Description: Checks for stubbed test subjects.
   Enabled: true
   VersionAdded: '1.7'
-  StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SubjectStub
+  StyleGuide: https://rspec.rubystyle.guide/#dont-stub-subject
```

Comment:
Sorry to bother, I guess this won't happen this time. It seems there's a script that switches it back to the old reference, and later fails if it's not as committed.

I'll handle that separately (https://github.com/rubocop-hq/rubocop-rspec/issues/1101).

Reply:
Ok. Reverting then 😄